src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java

```diff
@@ -17,21 +17,24 @@
 public final class CleanHelper {
 
 	private static final Logger LOG = LoggerFactory.getLogger(CleanHelper.class);
+	private static final String UNIX = "unix";
```

Comment:
I think the Cognifide standard is to include a blank line after field  so it's a bit easier to read it.